README.md

```diff
@@ -1,491 +1,20 @@
 # Ballerina MongoDB Connector
 
-Connects to MongoDB from ballerina 
+[![Build Status](https://github.com/ballerina-platform/module-ballerinax-mongodb/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-mongodb/actions?query=workflow%3ACI)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-mongodb.svg)](https://github.com/ballerina-platform/module-ballerinax-mongodb/commits/master)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-# Introduction
-## What is MongoDB?
 [MongoDB](https://docs.mongodb.com/v4.2/) is a general purpose, document-based, distributed database built for modern application developers and for the cloud era. MongoDB offers both a Community and an Enterprise version of the database.
 
-## Key Features of MongoDB
+Ballerina MongoDB connector connects to MongoDB from ballerina. 
 
-- List database names
-- List collection names
-- Insert
-- Update
-- Find
-- Delete
-- Count
+For more information, go to the module(s).
 
-## Connector Overview
+- [`ballerinax/mongodb`](https://docs.central.ballerina.io/ballerinax/mongodb/2.0.9)
```

Comment:
Please change the link without mentioning a version

Reply:
Fixed by 89976637b8fed8c9a16842370da109f71b035569.